### DISCUSSION TITLE
Expected Bool/Number/String/Timestamp, got ∅

### DISCUSSION THREAD

**Discussion by joanlloberacapo** [postmanlabs/postman-flows#327] [Defect]
Sometimes appears this message, also, when I try to set a current value to variable in a request, I cannot do it because durable variable says "Ambiguous ∅"

This problems appeared during this week.

**Comment by shrey-bhagat** [comment]
Hey @joanlloberacapo could you attach a screenshot as well for more clarity. But mostly this happens because the input port of this block is not connected to an output port of another block. The block takes input from other block(s) and shows it in durable and variables which you can use to set a value to a variable.

**Comment by joanlloberacapo** [comment]
Hi @shrey-bhagat, 

I can launch the workflow without problems, but this error persists. 
In this case, _lines_ is defined as an array, in other flows that I have don't show any warning/error. And the other attributes are well defined as string, but it's getting as a "any"

![image](https://user-images.githubusercontent.com/100215592/177932982-1183b9db-1207-43f6-8b0a-9696533bda30.png)
![image](https://user-images.githubusercontent.com/100215592/177933000-ea69170e-6d5f-4d99-80e9-ab5278853689.png)
![image](https://user-images.githubusercontent.com/100215592/177933049-771e3887-0c92-4b53-8e08-4f6a277c9c7e.png)
![image](https://user-images.githubusercontent.com/100215592/177933315-319aec01-e32c-4c83-aa26-d8c60bd6b33c.png)

**Comment by ghost** [comment]
Hey @joanlloberacapo,
- The example that you added for this request has the loop over a list or a tuple? (Check from the first request in the flow wherever you are looping)
- If the loop is over a tuple then for some reason the app is converting it's element to Ambiguous.
- If that is the case then edit the example to only include one element in the list so the app consider it as a list and then try it would take the current types.

Let me know if that works.

**Reply by joanlloberacapo** [reply]
It's over a list, and only one element. 
Not always fails, you could create another flow with the same flow from the beginning and it works. When you close the tab, and open again, sometimes you get errors, or sometimes fail other things

**Comment by FreddieSanchez** [comment]
I'm also seeing this issue. No looping in my example. 

`Request A` :heavy_check_mark: =>` Create Durable X` :heavy_check_mark: => `Request B` :heavy_check_mark:  => `Create Durable Y` :x: 

`Create Durable` Y, yields the error - the response is a very simple JSON response with 3 fields, none are null.

**Reply by FreddieSanchez** [reply]
I was able to solve the issue by creating an Example of `Request B`

**Reply by joanlloberacapo** [reply]
This problem seems to be solved automagically. Without changing values, now it works. 
Maybe some issues are resolved in any update
